.ddev/docker-compose.environment.yaml

```diff
@@ -0,0 +1,15 @@
+version: '3.6'
+
+services:
+    web:
+        environment:
+            - TYPO3_CONTEXT=Development
+            - TYPO3_INSTALL_DB_HOST=db
```

Comment:
@helhum Don't you see here a conflict by providing this TYPO3_INSTALL_* vars for testing? Perhaps we should do this by a .env-dist which the user can use or not. Context should not be a problem here imho